bibtex-actions.el

```diff
@@ -69,8 +70,15 @@
   "Face used to highlight content in `bibtex-actions' candidates."
   :group 'bibtex-actions)
 
+(defcustom bibtex-actions-bibliography (bibtex-actions--normalize-paths
+                                        bibtex-completion-bibliography)
```

Comment:
This caused a really hard to figure out problem for me where I would get `void function bibtex-actions--normalize-paths` type error. I fixed it by changing it to nil. I think it is an issue of order where this is evaluated before the definition of `bibtex-actions--normalize-paths` is.

Reply:
With this changed, this branch works for me

Reply:
So you think it's fine as is then, or is there something I should change?

BTW, I just pushed a fix for the above pdf/note issue. Was easy.

Reply:
> With this changed, this branch works for me

What do you mean by "this"?

Is this resolve, in other words?

Reply:
> > With this changed, this branch works for me
> 
> What do you mean by "this"?
> 
I am changing the default to `nil` fixed it for me. I think I have a proper fix. I will add a suggestion for it.

Reply:
Also I think I figured out the file-notify notes problem too. It was my mixing up local and global in the`unless` clauses in `bibtex-actions-filenotify-refresh`. I think exchanging those should fix it.

Reply:
> I don't think I can do this as a suggestion but I have all this on my branch and I think I can make pr to `csl-json` branch?

Yes, try a PR to this branch.

Reply:
Done.

Reply:
And see if it fixes the notes issue with filenotify watches too.

Reply:
Merged.